Sequencer/Sequencer/TcSequencer/Example/ExampleSeq1.TcPOU

```diff
@@ -0,0 +1,56 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<TcPlcObject Version="1.1.0.1">
+  <POU Name="ExampleSeq1" Id="{0b7420e2-7bee-0073-13f1-581e54ae5023}" SpecialFunc="None">
+    <Declaration><![CDATA[FUNCTION_BLOCK ExampleSeq1 EXTENDS Sequencer
+VAR_INPUT
+	em1 : IMOCK_EM;
+	em2 : IMOCK_EM;
+	em3 : IMOCK_EM;
+END_VAR]]></Declaration>
+    <Implementation>
+      <ST><![CDATA[CASE DoStep() OF
+
+	0:
+		IF OnEntry() THEN
+			RequestStep( 1 );			
```

Comment:
Can we see if STweep will standardize spaces around parameters automatically? This is inconsistent.